lib/mergometer/report.rb

```diff
@@ -46,13 +46,19 @@ def entries
       end
 
       def filter
-        "base:master repo:#{repo} type:pr is:merged"
+        "base:master #{repo_query} type:pr is:merged"
       end
 
       def fields_to_preload
         []
       end
 
+      def repo_query
+        @_query = repo.split(',').map do |r|
+          "repo:#{r} "
+        end.to_a.join
```

Comment:
Map already return an array I guess 😉

Reply:
Enumerator :(

Reply:
Can you double check?

```ruby
irb(main):002:0> ["1", "2"].map { |v| v }.join
=> "12"
```

Reply:
You are right, it seems I was using old ruby version before 🙇